### ISSUE TITLE
[BUG] Background colour of native date picker in form field and card

### ISSUE THREAD

**Issue by Hykalos** [kirbydesign/designsystem#3935]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When having a native date picker inside a `<kirby-form-field>` which is inside a `<kirby-card>`, the calendar icon's background is a darker grey than the rest of the input. 

## Describe how to reproduce the bug
1. Set up new angular project
2. Install Kirby
3. Set up dependencies
4. Insert the following in app.component.html:
```html
<kirby-card>
  <kirby-form-field label="Dato">
    <input
      kirby-input
      type="date"
      [useNativeDatePicker]="true"/>
  </kirby-form-field>
</kirby-card>
```
5. Open the page

## Which Kirby version was used?
10.5.1

## What was the expected behavior?
The colour of the icon's background is identical to the input's background colour.

## Add any screenshots

![Image](https://github.com/user-attachments/assets/2557e4fd-112d-4a92-958c-dc0fc335de4a)

## Please complete the following information:
- OS: Windows 11
- Browser Edge
- Browser version 136.0.3240.64

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
